src/test/java/com/exasol/adapter/dialects/saphana/util/dbbuilder/HanaSchema.java

```diff
@@ -0,0 +1,34 @@
+package com.exasol.adapter.dialects.saphana.util.dbbuilder;
```

Comment:
I was thinking if we should move this to the `test-db-builder-java`. Currently we have only one project involving Hana, so it isn't necessary immediately.

Reply:
👍